blockchain/contracts/PropertyOwnership.sol

```diff
@@ -14,7 +14,9 @@ contract PropertyOwnership is PropertyBase, ERC721Token {
   function buy(uint256 _propertyId) payable {
     require(msg.value >= properties[_propertyId].buy_price);
 
-    //
+    if (msg.value > properties[_propertyId].buy_price) {
```

Comment:
property buy price should be converted to wei here too!

Reply:
whoa, how is this even passing the tests